src/vs/workbench/services/languageDetection/browser/languageDetectionService.ts

```diff
@@ -168,6 +168,11 @@ export class LanguageDetectionService extends Disposable implements ILanguageDet
 	}
 }
 
-Registry.as<IWorkbenchContributionsRegistry>(Extensions.Workbench)
-	.registerWorkbenchContribution(LanguageDetectionService, LifecyclePhase.Eventually);
+export class LanguageDetectionWorkbenchContribution {
```

Comment:
Should this implement `IWorkbenchContribution`?

Reply:
`IWorkbenchContribution` is an empty interface. It's not needed.

Reply:
https://github.com/microsoft/vscode/blob/9350fa3c9df88b03860900d8fbe044544979ec26/src/vs/workbench/common/contributions.ts#L14-L16